make/data/cldr/common/properties/coverageLevels.txt

```diff
@@ -0,0 +1,141 @@
+#	coverageLevels.txt
+#	Copyright © 2022 Unicode, Inc.
```

Comment:
older version?

Reply:
is this from the alpha2 drop? or v42's?

Reply:
Yes, this is the file from the released CLDR v42. We have not integrated v43 yet.

Reply:
this is great groundwork.